io.catenax.manufactured_parts_quality_information/2.1.0/ManufacturedPartsQualityInformation.ttl

```diff
@@ -0,0 +1,248 @@
+
+:catenaXPartId a samm:Property ;
+   samm:preferredName "Catena-X Identifier of this part"@en ;
+   samm:description "The fully anonymous and unique Catena-X ID of this part. This id is unique inside Catena-X data space."@en ;
+   samm:characteristic ext-uuid:UuidV4Trait ;
+   samm:exampleValue "430f56d3-1234-1234-1234-efab12341234" .
+
+:catenaXQualityTaskId a samm:Property ;
+   samm:preferredName "Quality Task ID"@en ;
+   samm:description "The fully anonymous and unique Catena-X ID of this quality task. This id is unique inside Catena-X data space."@en ;
+   samm:characteristic ext-uuid:UuidV4Trait ;
+   samm:exampleValue "430f56d3-1234-1234-1234-aaaabbbbcccc" .
+
+:hasBeenReworked a samm:Property ;
+   samm:preferredName "Reworked"@en ;
+   samm:description "Indicator whether this part was reworked during manufacturing"@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue false .
+
+:manufacturerPartName a samm:Property ;
+   samm:preferredName "Manufacturer part name"@en ;
+   samm:description "Part name given from  the manufacturer or supplier of this part, component, sub component or raw material. A manufacturer part name is NOT unique: All parts of the same kind share the same part name."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Gear box housing sub-assembly" .
+
+:manufacturerPartNumber a samm:Property ;
+   samm:preferredName "Manufacturer part number"@en ;
+   samm:description "Part number given from  the manufacturer or supplier of this part, component, sub component or raw material. A manufacturer part number is NOT unique: All parts of the same kind share the same part number."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "FZ206460050202212" .
+
+:manufacturerSerialNumber a samm:Property ;
+   samm:preferredName "Manufacturer serial number"@en ;
+   samm:description "Serial number given from the manufacturer or supplier of this part, component, sub component. A manufacturer serial number is unique: With the serial number the produced part can be traced within the manufacturer."@en ;
+   samm:characteristic :UniqueID ;
+   samm:exampleValue "GBH3232434535454545-3534535" .
+
+:numberOfConductedEOLTests a samm:Property ;
+   samm:preferredName "Conducted EOL test"@en ;
+   samm:description "Number how often this part went through the EOL test"@en ;
+   samm:characteristic :PositiveNumber ;
+   samm:exampleValue "1"^^xsd:positiveInteger .
+
+:parentPartNumber a samm:Property ;
+   samm:preferredName "Parent part number"@en ;
+   samm:description "Internal part number of the analysed part.\nIf the manufacturer is a TIER1 supplier -> parent level is OEM.\nIf the manufacturer is a TIER2 supplier -> parent level is TIER1 supplier.\nA parent part number is NOT unique: All parent parts of the same kind share the same parent part number."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "12345" .
+
+:parentSerialNumber a samm:Property ;
```

Comment:
property naming changes required: 
`customerPartInstanceId` or
`parentPartInstanceId`

Reply:
As this model is used in a not Industry Core related use case, we decided to leave this as is for now. In a future version this should be resolved, which will, be guardrailed by a yet to define MS2 requirement. (#642 )